sys/include/phydat.h

```diff
@@ -79,6 +79,8 @@ enum {
     UNIT_TEMP_C,    /**< degree Celsius */
     UNIT_TEMP_F,    /**< degree Fahrenheit */
     UNIT_TEMP_K,    /**< Kelvin */
+    /* luminosity */
```

Comment:
I'm +1 @cgundogan comment and wikipedia [says the same](https://en.wikipedia.org/wiki/Illuminance). I pushed an update.